curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -243,17 +243,18 @@ int FileCacheManager::Write(uint64_t offset, uint64_t length,
     uint64_t chunkPos = offset % chunkSize;
     uint64_t writeLen = 0;
     uint64_t writeOffset = 0;
+    int64_t len = length;
```

Comment:
why change uint64_t to int64_t

Reply:
the code use the condition (length > 0) to break the loop, however the uint64_t is always >= 0

Reply:
> the code use the condition (length > 0) to break the loop, however the uint64_t is always >= 0 @ilixiaocui

`length = 0 `will break the loop.